crates/starknet-types-core/src/felt/mod.rs

```diff
@@ -1,9 +1,17 @@
+#[cfg(test)]
+mod felt_arbitrary;
+
 use core::ops::{Add, Mul, Neg};
 
 use bitvec::array::BitArray;
 use num_bigint::BigInt;
+
+use num_traits::{One, Zero};
```

Comment:
shouldn't this be behind a feature flag ?

Reply:
Here it's used not to be implemented on our Felt type, but to be able to call it's impl on `Bigint`.